system_test/common/helpers/aest_api.erl

```diff
@@ -56,6 +57,8 @@ sc_open(Params, Cfg) ->
     #{ pubkey := RPubKey, privkey := RPrivKey } = RAccount,
 
     {Host, _Port} = node_ws_int_addr(RNodeName, Cfg),
+    MaybeOpts = maps:merge(maybe_version_opts(Cfg),
+                           maybe_block_hash_opts(Params)),
```

Comment:
This fixes an issue with block hash deltas being passed as channel opening params. Old behaviour was ignoring those